hyrax.gemspec

```diff
@@ -84,10 +84,10 @@ EOF
   spec.add_development_dependency "equivalent-xml", '~> 0.5'
   spec.add_development_dependency "jasmine", '~> 2.3'
   spec.add_development_dependency 'rubocop', '~> 0.47.0'
-  spec.add_development_dependency 'rubocop-rspec', '~> 1.10.0'
+  spec.add_development_dependency 'rubocop-rspec', '~> 1.10', '~> 1.10.0'
```

Comment:
What is the effect of having two twiddle-wakkas on one line? I believe we want rubocop and rubocop-rspec pinned such that only patch-level updates come in.